test/unit/slc/utest_slc.py

```diff
@@ -152,7 +153,9 @@ def test_legal_names(name, slc_factory):
 
 
 @pytest.fixture
-def slc_with_tmp_checkout_dir(sample_slc_name, slc_factory) -> ScriptLanguageContainer:
+def slc_with_tmp_checkout_dir(
+    sample_slc_name, slc_factory
+) -> Iterator[ScriptLanguageContainer]:
     with slc_factory.context(slc_name=sample_slc_name, flavor="Vanilla") as slc:
```

Comment:
Is it actually an iterator?

Reply:
I think the fixture returns an instance of the `ScriptLanguageContainer`. The fact that it is implemented as a contextmanager is transparent for a consumer of this fixture.